analysis_scripts/analyze_mention_v_citation/mention_v_citation.md

```diff
@@ -0,0 +1,1080 @@
+    ## Table: mx Class Mention, top terms
+    ## 
+    ## |word           | class_m_count| class_all_count|     ratio|
+    ## |:--------------|-------------:|---------------:|---------:|
+    ## |bison          |            63|             132| 0.4772727|
+    ## |mexicos        |            55|             127| 0.4330709|
+    ## |mexican        |            96|             232| 0.4137931|
+    ## |rust           |            51|             181| 0.2817680|
+    ## |wheat          |           110|             476| 0.2310924|
+    ## |maize          |           119|             531| 0.2241055|
+    ## |maya           |            29|             133| 0.2180451|
+    ## |varieties      |            89|             485| 0.1835052|
+    ## |mexico         |           279|            1556| 0.1793059|
+    ## |monsanto       |            30|             169| 0.1775148|
+    ## |tolerant       |            23|             131| 0.1755725|
+    ## |nanotechnology |            52|             333| 0.1561562|
+    ## |autonomous     |            39|             259| 0.1505792|
+    ## |herbicide      |            19|             133| 0.1428571|
+    ## |transgenic     |            55|             505| 0.1089109|
+    ## 
+    ## 
+    ## Table: ph Class Mention, top terms
+    ## 
+    ## |word           | class_m_count| class_all_count|     ratio|
+    ## |:--------------|-------------:|---------------:|---------:|
+    ## |philippines    |            46|             175| 0.2628571|
+    ## |rice           |           161|             757| 0.2126816|
+    ## |geoengineering |            28|             210| 0.1333333|
+    ## |irrigation     |            18|             147| 0.1224490|
+    ## |rust           |            22|             181| 0.1215470|
+    ## |bt             |            27|             259| 0.1042471|
+    ## |cotton         |            28|             288| 0.0972222|
+    ## |taylor         |            34|             368| 0.0923913|
+    ## |monsanto       |            14|             169| 0.0828402|
+    ## |wheat          |            38|             476| 0.0798319|
+    ## |sheep          |            17|             226| 0.0752212|
+    ## |opera          |             8|             115| 0.0695652|
+    ## |tolerant       |             9|             131| 0.0687023|
+    ## |humanitarian   |             8|             119| 0.0672269|
+    ## |varieties      |            32|             485| 0.0659794|
+
+``` r
+mentions_freq = Reduce(merge, all_country_word_freq_m)
+mentions_freq$median_ratio = apply(mentions_freq[,2:ncol(mentions_freq)], 
+                                   1, median)
+mentions_freq = mentions_freq[order(mentions_freq$median_ratio, decreasing = T),]
+
+print(knitr::kable(head(mentions_freq,15), 
+                       caption = "Overall Class Mention, top terms, ratio is per country frequency"))
+```
+
+    ## 
+    ## 
+    ## Table: Overall Class Mention, top terms, ratio is per country frequency
```

Comment:
Ah, yes this is a good suggestion. The results are almost exactly the same, but this is much easier to understand since its ratios of counts instead of ratios of ratios.
added in https://github.com/greenelab/nature_news_disparities/pull/20/commits/9e3cd309d0ef5a2328ef222a74ae017e21ebac6c